Module testing/tesseract preprocessing deskew

urenajose opened the pull request:
Created a local env version of tesseract_preprocessing Notebook
conducted performance testing for deskew module.
https://github.com/sbrunner/deskew
Change images used for preprocessing for images with variety of different sizes

urenajose committed:
Create notebook with image preprocessing steps using
OpenCV, files used were sample data created by me to
explore Tesserac capabilites out of the box and with
minimum modification.

urenajose committed:
"Added a note to my code about files used and link
where to find them"

urenajose committed:
Created a local env version of
tesseract_preprocessing Notebook
Added perfomance testing for deskew module
Change images used for preprosessing
with images variety of different sizes

DevvinK reviewed (dismissed):
Simple and easy to read commit comments, but was there a way to upload the ipynb files is short bursts rather than the 1000 lines?

urenajose committed:
I just add it "This notebook it is meant to be use in google colab" to top be clear on utilization purpose.

ranaharoonkhan reviewed (dismissed):
I have reviewed the code looks great to me , really appreciate your efforts and hardwork for this project. Keep up the good work!

urenajose committed:
Added to tesseract_image_preprocessing_loca notebook:
code to count number of files in a directory to use to count images to be process by the system
clean the notebook of unecesary code and move all the dependencies to same cell

urenajose committed:
deleted some unecesary lines of code to count
number of files in a given folder

urenajose committed:
added for loop to print all the names of images
files to be preprocess

urenajose committed:
modify img_process function and  pre_process_image
return statments so they return the array instead of the save image.

urenajose committed:
Added variables for directories, parent directories
and abosulute path to current file.
Added Tesseract_temp_images directory
To keep the images that are temporary created
by the preprocesing functions

urenajose committed:
Created for loop to pre-process the images to be
use for training of  tesseract ouput custom traineddata
